python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -94,7 +94,7 @@ def runApPipeGen3(workspace, parsedCmdLine, processes=1):
     makeApdb(_getApdbArguments(workspace, parsedCmdLine))
 
     pipelineFile = _getPipelineFile(workspace, parsedCmdLine)
-    pipelineArgs = ["pipetask", "run",
+    pipelineArgs = ["pipetask", "--long-log", "run",
                     "--fail-fast",
```

Comment:
Preexisting, but can you add a comment that `--fail-fast` is to compensate for the extra interconnections added by `--graph-fixup`?